src/libs/auth.js

```diff
@@ -37,9 +39,8 @@ export const hasBillingScope = () => {
   return getAuthInstance().currentUser.get().hasGrantedScopes('https://www.googleapis.com/auth/cloud-billing')
 }
 
-const registeredAndAcceptedTos = (oldState, state) => {
-  return (oldState.registrationStatus !== 'registered' && state.registrationStatus === 'registered' && state.acceptedTos) ||
-    (state.registrationStatus === 'registered' && state.acceptedTos && !oldState.acceptedTos)
+const becameRegistered = (oldState, state) => {
+  return (oldState.registrationStatus !== userStatus.registeredWithTos && state.registrationStatus === userStatus.registeredWithTos)
```

Comment:
Nit: `newState` reads more easily to me for some reason:

```suggestion
const becameRegistered = (oldState, newState) => {
  return (oldState.registrationStatus !== userStatus.registeredWithTos && newState.registrationStatus === userStatus.registeredWithTos)
```

Reply:
Since the rest of this file uses `oldState` and `state`, I maintained it here for consistency.